extensions/parquet/table/src/main/java/io/deephaven/parquet/table/TypeInfos.java

```diff
@@ -93,9 +101,103 @@ private static TypeInfo lookupTypeInfo(
         return new ImmutablePair<>(SerializableCodec.class.getName(), null);
     }
 
+    static class PrecisionAndScale {
+        public final int precision;
+        public final int scale;
+
+        public PrecisionAndScale(final int precision, final int scale) {
+            this.precision = precision;
+            this.scale = scale;
+        }
+    }
+
+    private static PrecisionAndScale getPrecisionAndScale(final TrackingRowSet rowSet,
```

Comment:
I would call this `computePrecisionAndScale` to distinguish between it and the `getPrecisionAndScale` method.